lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -215,6 +216,36 @@ bool isAGatewayType(const std::string& mayaNodeType)
     return isInherited;
 }
 
+bool isMaterialsScope(const Ufe::SceneItem::Ptr& item)
```

Comment:
I feel like this should eventually go into `usdUfe/ufe/Utils.cpp`, but it currently still has a Maya dependency: `UsdMayaJobExportArgs::GetDefaultMaterialsScopeName()`. So, I think for now, `mayaUsd/ufe/Utils.cpp` makes more sense.